Assets/Packages/com.studio23.ss2.upmassistant/Editor/Manager/DataHandler.cs

```diff
@@ -0,0 +1,253 @@
+    public static readonly string PACKAGE_NAME = "packageNameData.json";
+   
+    public const string DefaultPackageName = "com.[companyname].[project].[packagename]";
+    public const string DefaultLicenceURL = "https://api.github.com/licenses/mit";//"https://api.github.com/licenses/lgpl-2.1";
+    
+    public static readonly string PACKAGE_JSON = "package.json";
+    public static readonly string README_MD = "README.md";
+    public static readonly string CHANGELOG_MD = "CHANGELOG.md";
+    public static readonly string LICENSE_MD = "LICENSE.md";
+    public static readonly string THIRDPARTYNOTICES_MD = "ThirdPartyNotices.md";
+     
+    
+    public static Dictionary<string, bool> FolderAndFilesList = new Dictionary<string, bool>
+    {
+        {PACKAGE_JSON, false},
+        {README_MD, false},
+        {CHANGELOG_MD, false},
+        {LICENSE_MD, false},
+        {THIRDPARTYNOTICES_MD, false},
+        {"Editor", false},
+        {$"Editor/[[packagename]].editor.asmdef", false},
+        {"Runtime", false},
+        {$"Runtime/[[packagename]].asmdef", false},
+        {"Runtime/Data", false},
+        {"Runtime/Core", false},
+        {"Tests", false},
+        {"Tests/EditMode", false},
+        {$"Tests/EditMode/editmode.tests.asmdef", false},
+        {"Tests/PlayMode", false},
+        {$"Tests/PlayMode/playmode.tests.asmdef", false},
+        {"Samples", false},
+        {"Documentation", false},
+           
+    };
+    public static void Initialized()
+    {
+        if (!Directory.Exists(ROOT))
+        {
+            Directory.CreateDirectory(ROOT);
+            Debug.Log("Folder structure created: " + ROOT);
+        }
+        if (!Directory.Exists(DATA_PATH))
+        {
+            Directory.CreateDirectory(DATA_PATH);
+            Debug.Log("Folder structure created: " + DATA_PATH);
+        } 
+        
+        var licensePath = Path.Combine(DATA_PATH, LICENSE_URL);
+        if (!File.Exists(licensePath))
+        {
+            LicenseURLData data = new LicenseURLData { LicenseURL = DefaultLicenceURL };
+            string jsonData = JsonUtility.ToJson(data);
+            File.WriteAllText(licensePath, jsonData);
+            // File.Create(licensePath).Close();
+        }
+        
+        var packageNamePath = Path.Combine(DATA_PATH, PACKAGE_NAME);
+        if (!File.Exists(packageNamePath))
+        {
+            string updatedPackageName = DefaultPackageName;
+            updatedPackageName = updatedPackageName.Replace("[companyname]", PlayerSettings.companyName);
+            updatedPackageName = updatedPackageName.Replace("[project]", "ss2"); //Application.productName
+            updatedPackageName = updatedPackageName.Replace("[packagename]", PlayerSettings.productName);
+            updatedPackageName = updatedPackageName.Replace(" ", "").ToLower();
+            PackageNameData data = new PackageNameData { PackageName = updatedPackageName };
+            string jsonData = JsonUtility.ToJson(data);
```

Comment:
Use JsonConvert.Serialize() from newtonsoft instead of using jsonutility for consistency.